### ISSUE TITLE
bypass_state discussion

### ISSUE THREAD

**Issue by svde** [vekexasia/comfoair-esp32#6]
Not really an issue, but more something I'd like to discuss.

I noticed that bypass_state is treated like on/off. I reality the bypass valve can be opened partially. I'm also seeing that from candump:

  can0  0038C041   [1]  64
  can0  0038C041   [1]  **5D**
  can0  0038C041   [1]  64
  can0  0038C041   [1]  00
  can0  0038C041   [1]  64
  can0  0038C041   [1]  00
  can0  0038C041   [1]  64
  can0  0038C041   [1]  00

Would it be possible to implement it as a percentage instead?
```
-      LAZYSWITCH(227, "bypass_state", "%s", vals[0] == 64 ? "on" : "off")  // 1 = on, 0 = off   
+      LAZYSWITCH(227, "bypass_state", "%d", vals[0])  // %
```

**Comment by vekexasia** [comment]
hey @svde why not. i guess so. anyhow i still have to understand what is the bypass_state exactly.

**Comment by svde** [comment]
OK, it's included in #7
